### ISSUE TITLE
InfoBar with just Content isn't displayed as expected

### ISSUE THREAD

**Issue by michael-hawker** [microsoft/microsoft-ui-xaml#8730]
### Describe the bug

If you have an InfoBar set with just Content and no `Title` or `Message` it isn't displayed as one would expect:

![image](https://github.com/microsoft/microsoft-ui-xaml/assets/24302614/2998c6f4-125f-497e-bb5a-d74c22b8775c)

You may want to use `Content` to display a variety of things either some other UIElement or maybe just a simple text message. If this is done programmatically, it's more complex to have to do logic to set a Title/Message and/or Content then just use the simple single `Content` property.

### Steps to reproduce the bug

```xml
    <Grid Padding="40">
        <muxc:InfoBar IsOpen="True" Content="Short Message"/>
    </Grid>
```

### Expected behavior

No extra gap on top when Content is set and there is no defined Title/Message:

![image](https://github.com/microsoft/microsoft-ui-xaml/assets/24302614/b1af7fd4-ad22-4ef9-ad79-b713d6d0a0fe)

### NuGet package version

None

### Additional context

Occurs on both WinUI 2 and 3

**Comment by JJBrychell** [comment]
Info bar uses a 3x2 grid to display its contents.  The top row is the icon, the title/message/action and the close bar.  While the bottom row is just to content in the second column.  Hence when there is no title/message/action that cell is left blank and result looks odd. 

The fix is in to move the content to the first row if there is no title/message/action and scheduled for 1.5.

**Comment by michael-hawker** [comment]
Thanks @JJBrychell will this update be backported to WinUI 2 as well? Has a fix shipped in a 1.5 preview that can be validated?

I believe in the past issues were closed when they shipped as well alongside the build numbers with a bot, did that change?
